### PR TITLE
Add `explore images` icon

### LINKED ISSUE #8862: [Icon]: New icon <ExploreImages>
### How will these icons be used?

This icon was designed for the new action **Explore free images** in the image banner menu. This action directs the user to Burst stock imagery, which is a library of free images can be used by the merchants to build their online store.
New-Icon

In Context
Screenshot 2023-03-31 at 1 58 08 PM

### Icon name

ExploreImages 

### What type of icon set?

Major (20x20)

### Icon description

Action: Used to signify exploring free images 
Wayfinding: Used to help direct merchants to attach and add free images to their store
Concept: represents exploring a variety of image categories

### Icon keywords

Images, media

### Icon author

Osama ghazal

### Icon SVG code

```shell
<svg  viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M14.3846 11C12.5153 11 11 12.5153 11 14.3846C11 16.2539 12.5153 17.7692 14.3846 17.7692C14.9838 17.7692 15.5467 17.6135 16.0349 17.3403L17.4242 18.7296C17.7847 19.0901 18.3692 19.0901 18.7296 18.7296C19.0901 18.3692 19.0901 17.7847 18.7296 17.4242L17.3403 16.0349C17.6135 15.5467 17.7692 14.9838 17.7692 14.3846C17.7692 12.5153 16.2539 11 14.3846 11ZM12.8462 14.3846C12.8462 13.5349 13.5349 12.8462 14.3846 12.8462C15.2343 12.8462 15.9231 13.5349 15.9231 14.3846C15.9231 15.2343 15.2343 15.9231 14.3846 15.9231C13.5349 15.9231 12.8462 15.2343 12.8462 14.3846Z" fill="#5C5F62"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M0 1.48657C0 0.665562 0.665563 0 1.48658 0H16.3523C17.1733 0 17.8389 0.665562 17.8389 1.48658V9.71496C16.9643 9.20902 15.9488 8.91945 14.8658 8.91945C14.028 8.91945 13.2308 9.09268 12.5079 9.40527L11.7935 8.16526C11.6052 7.85804 11.1691 7.84813 10.961 8.14544L7.9284 12.8837L6.31299 11.0492C6.11478 10.851 5.78773 10.851 5.59943 11.0591L2.07823 15.0739C1.84038 15.4009 2.06832 15.8568 2.47465 15.8568H9.00167C9.12103 16.5683 9.36675 17.237 9.71496 17.8389H1.48658C0.665563 17.8389 0 17.1733 0 16.3523V1.48657ZM8.42393 5.45078C8.42393 4.36062 7.53198 3.46868 6.44183 3.46868C5.35167 3.46868 4.45973 4.36062 4.45973 5.45078C4.45973 6.54093 5.35167 7.43288 6.44183 7.43288C7.53198 7.43288 8.42393 6.54093 8.42393 5.45078Z" fill="#5C5F62"/>
</svg>
```

### Are there any other important details?

Here's the [pull request](https://github.com/Shopify/polaris/pull/8861) we've created for the it

### PR DESCRIPTION
**Pull request by AlejandroE**
### WHY are these changes introduced?

closes: https://github.com/Shopify/polaris/issues/8862

Adds a new icon to represent the menu action opening free (burst) images in OSE

New-Icon

### 🎩 checklist

- [x] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [x] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [x] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [x] Updated the component's `README.md` with documentation changes
- [x] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

**Comment by AlejandroE** [comment]
CC @osamaghazal01

**Comment by osamaghazal01** [comment]
Solves this issue
https://github.com/Shopify/polaris/issues/8862

**Review by martenbjork — CHANGES_REQUESTED** [review]
Looks great!

The icon description in the `yml` shouldn't be a Markdown table. It should just be a simple string. (I saw that `AddImageMajor` icon had a commented out Markdown table, but that's incorrect — sorry about the confusion!)

**Review by martenbjork — APPROVED** [review]

**Comment by heyjoethomas** [comment]
Please make sure to add this into the Figma UIKit as well. If you need help or have questions on how to do that feel free to reach out.

**Comment by AlejandroE** [comment]
@heyjoethomas thanks for the heads up! I'm approaching the end of my day but will figure that out tomorrow :D

**Comment by osamaghazal01** [comment]
@heyjoethomas I have access to [this kit](https://www.figma.com/file/mMHFt3kEDNjLMZWowi6gnt/Polaris-Icons?node-id=6%3A10&t=iPLxoBIoVKGvvo8B-1) but I don't have access to edit.

**Comment by heyjoethomas** [comment]
> @heyjoethomas I have access to [this kit](https://www.figma.com/file/mMHFt3kEDNjLMZWowi6gnt/Polaris-Icons?node-id=6%3A10&t=iPLxoBIoVKGvvo8B-1) but I don't have access to edit.

You can create a branch to add the icon. Then just request a review when ready and I can merge and publish.

File: .changeset/poor-walls-care.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': major
+---
+
+Adds a new icon used to represent exploring a variety of imagery assets
```
Comment (kyledurand) [inline .changeset/poor-walls-care.md]:
```suggestion
Added ExploreImagesMajor icon
```

File: .changeset/poor-walls-care.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': major
```
Comment (kyledurand) [inline .changeset/poor-walls-care.md]:
```suggestion
'@shopify/polaris-icons': minor
```

**Review by kyledurand — CHANGES_REQUESTED** [review]
Just make sure to change from major to minor release

**Comment by AlejandroE** [comment]
@kyledurand thanks Kyle! made the changes but the checks seem to be flaky, might be github.

**Review by kyledurand — APPROVED** [review]
Code looks good and I pushed up some changes removing extraneous attributes that were causing tests to fail. There may be a chance we'll need to skip the comment check if that still fails. Seems like a false positive. 

Last thing I'm seeing is that the icon isn't centered in its canvas. I'm assuming that's intentional but wanted to flag it just in case

image

**Comment by osamaghazal01** [comment]
Thank you folks :)

**Pull request merged**